mpf/platforms/fast/fast.py

```diff
@@ -859,8 +859,15 @@ def clear_hw_rule(self, switch, coil):
     def receive_bootloader(self, msg, remote_processor):
         """Process bootloader message."""
         self.debug_log("Got Bootloader message: %s from", msg, remote_processor)
+        ignore_rgb = self.config['ignore_rgb_crash'] and \
+            remote_processor == self.rgb_connection.remote_processor
         if msg in ('00', '02'):
-            self.error_log("The FAST %s processor rebooted. Unfortunately, that means that it lost all it's state "
+            self.error_log("The FAST {} processor rebooted. Unfortunately, that means that it lost all it's state "
```

Comment:
I think the convention for log lines is still the old percentage syntax.

Reply:
I saw that, but wasn't sure if that was by design or just legacy code. I'm in favor up updating old syntax as files are updated, but I'm happy to stick with the old style if that's desirable.

Reply:
The reason for the older syntax here is that it will can be lazy evaluated. If you use format we will always replace those strings even when logging is disabled. With percentage syntax replacement will happen only if the string is displayed. The linter even complains about this when you use logger.debug but since we wrapped it this is not happening here.